### PR TITLE
FOGL-5144: handling of bearer token - security category - sendToDispatcher

### PR DESCRIPTION
**Pull request by pintomax**

### CODE REVIEW

**Review by MarkRiddoch — COMMENTED** [review]
I don't see a ServiceAuthHandler class in this PR, is this something you need to add?

**Comment by pintomax** [comment]
This PR needs Fledge branch "FOGL-5144_bearer_token_verification"

File: C/services/common/include/notification_service.h [hunk 1]
```diff
@@ -42,20 +43,17 @@ class NotificationService : public ServiceHandler
...
 
 	private:
-		const std::string	m_name;
 		Logger*			m_logger;
 		bool			m_shutdown;
-		NotificationApi*	m_api;
-		ManagementClient* 	m_managerClient;
-		ManagementApi*		m_managementApi;
-		StorageClient*		m_storage;
+		NotificationApi*  m_api;
```
Comment (MarkRiddoch) [inline C/services/common/include/notification_service.h:51]: Indentation

**Review by MarkRiddoch — CHANGES_REQUESTED** [review]
Please preserve the indentation in the header file

**Review by MarkRiddoch — DISMISSED** [review]

**Review by MarkRiddoch — APPROVED** [review]

**Review by MarkRiddoch — DISMISSED** [review]

**Review by MarkRiddoch — DISMISSED** [review]

**Review by MarkRiddoch — APPROVED** [review]

**Pull request merged**